src/app/components/ATIAnalytics/params/genericPage/buildParams.ts

```diff
@@ -27,6 +27,7 @@ export const buildPageATIParams = ({
     timeUpdated,
   } = atiData;
 
+  const ampExperimentName = atiData?.ampExperimentName;
```

Comment:
**Question** Is atiData ever undefined? The types suggest not, so the chaining should not be necesary.

Reply:
Yeah I'm not sure what I was trying to do there, put it in with the rest of them above